app/views/inventories/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>My Inventories</h1>
+
+<% @inventories.each do |inventory| %>
+  <div>
+    <%= link_to inventory.name, inventory %>
+    <%= button_to "Delete", inventory, method: :delete, data: { confirm: "Are you sure you want to delete this inventory?" } %>
+  </div>
+<% end %>
+
+<%= link_to "New Inventory", new_inventory_path %>
```

Comment:
- You have done well so far but this page does not match the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/inventory_index.png) as it is in the project requirements. Please correct this 👍 👍.
<img width="953" alt="inent" src="https://github.com/Anand8317/recipe/assets/73607512/5a474bea-36cb-4a12-8eee-caaad5957a63">

Reply:
- Well resolved 
<img width="1073" alt="Screenshot 2024-02-18 at 00 53 33" src="https://github.com/Anand8317/recipe/assets/58692590/506770aa-a411-4437-a778-16a0637fecd6">